src/utils/store/store-initializer.ts

```diff
@@ -16,11 +21,27 @@ type CreateStoreInitializerArgs<T extends object> = {
   enhancers?: Array<StoreEnhancer>;
 };
 
+/**
+ * Util that used for creating the application's initStore.
+ *
+ * @template State - The type of the application state.
+ * @param {CreateStoreInitializerArgs<State>} args - The arguments for creating the store.
+ * @param {Reducer<State>} args.rootReducer - The root reducer function.
+ * @param {Array<Middleware>} [args.middlewares] - The middlewares for the store.
+ * @param {Array<StoreEnhancer>} [args.enhancers] - The enhancers for the store.
```

Comment:
I offer to show the default values 

https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

```suggestion
 * @param {Array<Middleware>} [args.middlewares=[]] - The middlewares for the store.
 * @param {Array<StoreEnhancer>} [args.enhancers=[]] - The enhancers for the store.
```